advanced/Scripts/piholeDebug.sh

```diff
@@ -394,41 +394,46 @@ os_check() {
     # Extract dig response
     response="${cmdResult%%$'\n'*}"
 
-    IFS=" " read -r -a supportedOS < <(echo "${response}" | tr -d '"')
-    for distro_and_versions in "${supportedOS[@]}"
-    do
-        distro_part="${distro_and_versions%%=*}"
-        versions_part="${distro_and_versions##*=}"
-
-        if [[ "${detected_os^^}" =~ ${distro_part^^} ]]; then
-            valid_os=true
-            IFS="," read -r -a supportedVer <<<"${versions_part}"
-            for version in "${supportedVer[@]}"
-            do
-                if [[ "${detected_version}" =~ $version ]]; then
-                    valid_version=true
-                    break
-                fi
-            done
-            break
-        fi
-    done
-
     log_write "${INFO} dig return code:  ${digReturnCode}"
     log_write "${INFO} dig response:  ${response}"
 
-    if [ "$valid_os" = true ]; then
-        log_write "${TICK} Distro:  ${COL_GREEN}${detected_os^}${COL_NC}"
+    if [ "${response}" -ne 0 ]; then
+        log_write "${CROSS} Distro: ${COL_RED}${detected_os^}${COL_NC}"
```

Comment:
```suggestion
        log_write "${INFO} Distro: ${COL_RED}${detected_os^}${COL_NC}"
        log_write "${INFO} Version: ${COL_GREEN}${detected_version}${COL_NC}"
```

Maybe info instead of cross, as we have successfully determined the OS (and version) we just have nothing to compare it to.